error/s2n_errno.h

```diff
@@ -310,28 +310,49 @@ typedef enum {
 #define S2N_DEBUG_STR_LEN 128
 extern __thread const char *s2n_debug_str;
 
-#define TO_STRING(s) #s
-#define STRING_(s) TO_STRING(s)
+#define TO_STRING(s)   #s
+#define STRING_(s)     TO_STRING(s)
 #define STRING__LINE__ STRING_(__LINE__)
 
-#define _S2N_DEBUG_LINE     "Error encountered in " __FILE__ ":" STRING__LINE__
-#define _S2N_ERROR( x )     do { s2n_debug_str = _S2N_DEBUG_LINE; s2n_errno = ( x ); s2n_calculate_stacktrace(); } while (0)
-#define S2N_ERROR_PRESERVE_ERRNO() do { return -1; } while (0)
-#define S2N_ERROR_IS_BLOCKING( x )    ( s2n_error_get_type(x) == S2N_ERR_T_BLOCKED )
+#define _S2N_DEBUG_LINE "Error encountered in " __FILE__ ":" STRING__LINE__
+#define _S2N_ERROR(x)                    \
+    do {                                 \
+        s2n_debug_str = _S2N_DEBUG_LINE; \
+        s2n_errno = (x);                 \
+        s2n_calculate_stacktrace();      \
+    } while (0)
+#define S2N_ERROR_PRESERVE_ERRNO() \
+    do {                           \
+        return -1;                 \
+    } while (0)
+#define S2N_ERROR_IS_BLOCKING(x) (s2n_error_get_type(x) == S2N_ERR_T_BLOCKED)
 
 /* DEPRECATED: use POSIX_BAIL instead */
-#define S2N_ERROR( x )      do { _S2N_ERROR( ( x ) ); return -1; } while (0)
+#define S2N_ERROR(x)     \
+    do {                 \
+        _S2N_ERROR((x)); \
+        return -1;       \
+    } while (0)
 
 /* DEPRECATED: use PTR_BAIL instead */
-#define S2N_ERROR_PTR( x )  do { _S2N_ERROR( ( x ) ); return NULL; } while (0)
+#define S2N_ERROR_PTR(x) \
+    do {                 \
+        _S2N_ERROR((x)); \
+        return NULL;     \
+    } while (0)
 
 /* DEPRECATED: use POSIX_ENSURE instead */
-#define S2N_ERROR_IF( cond , x ) do { if ( cond ) { S2N_ERROR( x ); }} while (0)
+#define S2N_ERROR_IF(cond, x) \
+    do {                      \
+        if (cond) {           \
+            S2N_ERROR(x);     \
+        }                     \
+    } while (0)
 
 /** Calculate and print stacktraces */
 struct s2n_stacktrace {
-  char **trace;
-  int trace_size;
+    char **trace;
+    int trace_size;
```

Comment:
I can't comment on it, but there's a "clang-format on" at the end of this file that's no longer needed.

I'd just ctrl+f this directory for all the "clang-formats" and make sure they make sense.